external_libraries/CMakeLists.txt

```diff
@@ -82,40 +82,43 @@ if(LTO)
 				 APPEND PROPERTY LINK_FLAGS "-flto -flto-report")
 endif()
 
-if(NOT YAMLCPP_FOUND)
-  message(STATUS "using bundled libyaml-cpp")
+if(SC_SCLANG)
```

Comment:
easier: use `add_library( ... EXCLUDE_FROM_ALL )` ... and the libraries will only be built when they are needed as dependency of another target.

compare: https://cmake.org/cmake/help/v3.4/command/add_library.html